images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -331,6 +331,20 @@ function Get-DotNetCoreSdkVersions {
     return $dotNetCoreSdkVersion
 }
 
+function Get-DotnetTools {
```

Comment:
We should return values from the function. Currently, we just add them to the  $toolsList array.